src/main/java/com/kodilla/ecommercee/dto/ProductDto.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.ecommercee.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ProductDto {
+
+    private  int id;
+
+    private String name;
+
+    private String description;
+
+    private double price;
+
+    private int groupId;
+
```

Comment:
Zbędne puste linie na końcu i między polami klasy.

Reply:
dzięki za code review , biorę sie za poprawe.
Hmm ale ciekawe ponieważ u mnie zarówno przechodzą wszystkie testy jak i buduje się projekt. Jaki błąd Ci się generuje Andrzej ?

Reply:
Muszę zweryfikować czy to nie jest błąd konfiguracji CircleCI - na razie zignoruj :)